roles/openshift_node/templates/partials/kubeletArguments.j2

```diff
@@ -1,4 +1,4 @@
-{% if openshift.common.use_cluster_metrics | bool %}
+{% if openshift.common.use_cluster_metrics | bool or openshift.common.use_cluster_metrics_hawkular | bool %}
```

Comment:
The read-only port should not be needed for hawkular, it was a workaround needed for the previous metrics role.